### PR TITLE
Add saving, swapping ledger scripts and sample programs deployed ledger for fast boot

### PR DESCRIPTION
**Pull request by fulltimemike**
I have a sample ledger folder that can be hot swapped into the local aleo network. We can get rid of that, but it might be nice to build some sample ledger starts that make it easy for reproducible states to be shared.

### CODE REVIEW

**Comment by fulltimemike** [comment]
Basically, this saves the `.ledger*` files, and then clears `.ledger*` and `.currrent*` files and provides seeded `.ledger*` files. RPC still has to be reset so it's not quite unit testing yet

File: pondo-bot/.env.example [hunk 1]
```diff
@@ -24,3 +24,5 @@ TEST_USER1_PRIVATE_KEY=test_user1_private_key
 TEST_USER1_ADDRESS=test_user1_address
 TEST_USER2_PRIVATE_KEY=test_user2_private_key
 TEST_USER2_ADDRESS=test_user2_address
+LEDGER_PATH=/Users/you/snarkOS/
```
Comment (evanmarshall) [inline pondo-bot/.env.example:27]: Isn't the ledger path whenever you run ./devnet ? So if you start it from within a script here, wouldn't it just place those files wherever and you wouldn't have to worry about copying them or where they are in general?

**Review by evanmarshall — COMMENTED** [review]
We probably shouldn't commit any of the `.ledger` files

File: pondo-bot/package.json [hunk 2]
```diff
@@ -14,7 +14,9 @@
     "call": "yarn call:protocol",
     "runOracleTests": "yarn build:dev && node ./dist/runOracleTests.js",
     "call:test": "yarn build:dev && node ./dist/testProgramCalls.js",
-    "getTokenOwnerHash": "node ./dist/getTokenOwnerHash.js"
+    "getTokenOwnerHash": "node ./dist/getTokenOwnerHash.js",
+    "snapshotLedger": "node ./dist/snapshotLedger.js",
```
Comment (evanmarshall) [inline pondo-bot/package.json:18]: Ideally, we'd start the `./devnet` & `rpc` from here too.

**Pull request closed without merging**